server/routes/bitbucket/bitbucket_selection.py

```
@@ -76,15 +163,78 @@ def save_workspace_selection(user_id):
+            with conn.cursor() as cur:
+                set_rls_context(cur, conn, user_id, log_prefix="[BitbucketMetadata:update]")
+                cur.execute(
+                    """UPDATE connected_repos
+                       SET metadata_summary = %s, metadata_status = 'ready', updated_at = NOW()
+                       WHERE provider = 'bitbucket' AND repo_full_name = %s AND user_id = %s""",
+                    (summary, repo_full_name, user_id),
+                )
+                conn.commit()
+        return jsonify({"message": "Metadata updated"})
+    except Exception as e:
+        logger.error(f"Error updating repo metadata: {e}", exc_info=True)
```

Ruff (0.15.15)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)